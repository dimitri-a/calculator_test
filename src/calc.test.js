import React from 'react';
import renderer from 'react-test-renderer';
import ReactTestUtils from 'react-dom/test-utils'
import Calculator from "./calculator";

test('test that calculator snapshot', () => {
    const component = renderer.create(
        <Calculator></Calculator>
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();

    let res = component.getInstance().calcSum(122,2);
    


});test('test that calculator again sum', () => {
    const component = renderer.create(
        <Calculator></Calculator>
    );
    let tree = component.toJSON();

    let res = component.getInstance().calcSum(122,2);

    console.log('res=',res);

    expect(res).toEqual(124);


});