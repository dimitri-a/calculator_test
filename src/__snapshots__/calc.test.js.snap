// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test that calculator snapshot 1`] = `
<div>
  <input
    placeholder="number 1"
    type="text"
  />
  <input
    placeholder="number 2"
    type="text"
  />
  <button
    onClick={NaN}
  >
    sum
  </button>
  sum: 
  0
</div>
`;
